### ISSUE TITLE
Add capability for users to create groups and share resources with groups

### ISSUE THREAD

**Issue by dtarb** [hydroshare/hydroshare#14]
Group functionality is outlined in the Access Control Policy document in Alfresco https://my.alfresco.com/share/s/WJuMf2ghRlyuty0oXrqe5g.

**Comment by horsburgh** [comment]
I think we should stick with all resource creation done at the individual user level.  Resources should then be shared with groups.  Groups cannot create resources.

**Comment by dtarb** [comment]
@horsburgh is correct.  Sorry if my title a year ago was ambiguous.  I've edited the title to be a bit better .  Users create resources.  Users also create groups.  Groups cannot create resources but resources can be shared with groups at either the edit or view level.  This is detailed in the access control document linked above.  I've also edited out the statements about iRODS rules and policy enforcement points since access control has been implemented differently.
